### ISSUE TITLE
You are not authorized to access SendGrid, please contact Support.

### ISSUE THREAD

**Issue by Abraao-Andrad** [sendgrid/sendgrid-python#1053]
You are not authorized to access SendGrid, please contact [Support](https://support.sendgrid.com/).

I'm having problems accessing my account, I've already tried to follow the step by step of accessing support and going to I have a different problem with logging in but to access this page they ask you to log in to the platform which is precisely my initial problem.

**Comment by josefinarcasanova** [comment]
Hi! I'm having the exact same problem. And cannot contact Support since it also asks me to log in first (oh, the irony!). So yeah, if anyone from SendGrid would help, I'd appreciate it.

**Comment by Michael-Vol** [comment]
Got same issue here.

**Comment by MXakon** [comment]
yep, I also encountered this problem. After successful registration, I can't log in. I click support but choose to log in before sending?

**Comment by tonytromp02** [comment]
Seems the only way is MailChimp. What a bummer
